.github/workflows/calibreapp-image-actions.yml

```diff
@@ -9,7 +9,7 @@ jobs:
         uses: actions/checkout@master
 
       - name: Compress Images
-        uses: calibreapp/image-actions@master
+        uses: calibreapp/image-actions@main
```

Comment:
To give a litle bit of context I'll leave a link to the slack convo that suggested this change https://meli.slack.com/archives/CSKLKAGC8/p1643735490929499

Reply:
adjusted